In [1]:
from scipy import stats

In [2]:
# For x>44

z=(44-38)/6
z

1.0

In [3]:
#Employees older than 44 years of age ,p(x>44)
P_Greater_44 = 1-stats.norm.cdf(abs(z))
P_Greater_44

0.15865525393145707

In [4]:
#Employees between 38 and 44 years of age using formula stats.norm.cdf(44, loc=38, scale=6)

Bet_38_44 = stats.norm.cdf(44, 38,6)-stats.norm.cdf(38,38,6)
Bet_38_44

0.3413447460685429

A. More employees at the processing center are older than 44 than between 38 and 44. ***is False*** as value calculated in line 3 and line 4 shows that employees between 38 and 44 years of age is more.

In [6]:
#Number of employees attending training under the age of 30 years
Under_30 = 400 * stats.norm.cdf(30,38,6)
Under_30

36.484487890347154

B.	A training program for employees under the age of 30 at the center would be expected to attract about 36 employees.
The above statement ***is true*** as per the calculation in line number 9 and output 9 is 36.48.